src/Resources/config/routing/api.xml

```diff
@@ -1,39 +1,70 @@
         <requirement key="_format">json|xml|html</requirement>
     </route>
-    <route id="sonata_api_user_user_delete_user_group" path="/user/{userId}/{groupId}.{_format}" methods="DELETE" controller="Sonata\UserBundle\Controller\Api\Legacy\UserController::deleteUserGroupAction" format="json">
+    <route id="sonata_api_user_user_delete_user_group" path="/users/{userId}/groups/{groupId}.{_format}" methods="DELETE" controller="Sonata\UserBundle\Controller\Api\Legacy\UserController::deleteUserGroupAction" format="json">
         <requirement key="_format">json|xml|html</requirement>
     </route>
-    <route id="sonata_api_user_user_get_user" path="/user/{id}.{_format}" methods="GET" controller="Sonata\UserBundle\Controller\Api\Legacy\UserController::getUserAction" format="json">
+    <route id="sonata_api_user_user_get_user" path="/users/{id}.{_format}" methods="GET" controller="Sonata\UserBundle\Controller\Api\Legacy\UserController::getUserAction" format="json">
         <requirement key="_format">json|xml|html</requirement>
     </route>
     <route id="sonata_api_user_user_get_users" path="/users.{_format}" methods="GET" controller="Sonata\UserBundle\Controller\Api\Legacy\UserController::getUsersAction" format="json">
         <requirement key="_format">json|xml|html</requirement>
     </route>
-    <route id="sonata_api_user_user_post_user" path="/user.{_format}" methods="POST" controller="Sonata\UserBundle\Controller\Api\Legacy\UserController::postUsersAction" format="json">
+    <route id="sonata_api_user_user_post_user" path="/users.{_format}" methods="POST" controller="Sonata\UserBundle\Controller\Api\Legacy\UserController::putUserAction" format="json">
         <requirement key="_format">json|xml|html</requirement>
     </route>
-    <route id="sonata_api_user_user_post_user_group" path="/user/{userId}/{groupId}.{_format}" methods="POST" controller="Sonata\UserBundle\Controller\Legacy\Api\UserController::postUserGroupAction" format="json">
+    <route id="sonata_api_user_user_post_user_group" path="/users/{userId}/groups/{groupId}.{_format}" methods="POST" controller="Sonata\UserBundle\Controller\Legacy\Api\UserController::postUserGroupAction" format="json">
         <requirement key="_format">json|xml|html</requirement>
     </route>
-    <route id="sonata_api_user_user_put_user" path="/user/{id}.{_format}" methods="PUT" controller="Sonata\UserBundle\Controller\Api\Legacy\UserController::putUsersAction" format="json">
+    <route id="sonata_api_user_user_put_user" path="/users/{id}.{_format}" methods="PUT" controller="Sonata\UserBundle\Controller\Api\Legacy\UserController::putUserAction" format="json">
+        <requirement key="_format">json|xml|html</requirement>
+    </route>
+    <!-- wrong routes introduce in version 4.6.0 - keep for BC -->
```

Comment:
We need a NEXT_MAJOR comment to removing them ?

Reply:
we don't. In NEXT_MAJOR we will remove support for `nelmio/api-doc-bundle` v2. `api.xml` will be replaced by `api_nelmio_v3.xml`.